Unity-Technologies/Graphics
# Fix compilation error when VR module is disabled in the built-in Unity modules

### Purpose of this PR
Fix compilation error when VR module is disabled in the built-in Unity modules.

---
### Testing status
Using the repro steps in the bug report: https://fogbugz.unity3d.com/f/cases/1389160/
1. Create a new project
2. Disable VR Built-in package
3. Install Post Processing package
4. Observe the Console output

Tested compilation of the package when disabling built-in VR module in Package Manager (and with VR module re-activated afterward)
